### PR TITLE
Fix component governance bugs

### PR DESCRIPTION
**Pull request by wtgodbe**
Part of dotnet/aspnetcore#37889

### CODE REVIEW

File: src/Microsoft.AspNet.WebHooks.Custom.AzureStorage/packages.config [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <package id="Microsoft.AspNetCore.Http.Abstractions" version="1.0.0" targetFramework="net451" />
   <package id="Microsoft.AspNetCore.Http.Features" version="1.0.0" targetFramework="net451" />
   <package id="Microsoft.Data.Edm" version="5.6.2" targetFramework="net45" />
-  <package id="Microsoft.Data.OData" version="5.6.2" targetFramework="net45" />
+  <package id="Microsoft.Data.OData" version="5.8.4" targetFramework="net452" />
```
Comment (Pilchie) [inline src/Microsoft.AspNet.WebHooks.Custom.AzureStorage/packages.config:13]: L12 of the corresponding .csproj file says this project targets .NET Framework 4.5.1.  Is this a problem?

File: test/Microsoft.AspNet.WebHooks.Custom.AzureStorage.Test/packages.config [hunk 2]
```diff
@@ -3,7 +3,7 @@
   <package id="Microsoft.AspNet.WebApi.Client" version="5.2.2" targetFramework="net451" />
   <package id="Microsoft.AspNet.WebApi.Core" version="5.2.2" targetFramework="net451" />
   <package id="Microsoft.Data.Edm" version="5.6.2" targetFramework="net45" />
-  <package id="Microsoft.Data.OData" version="5.6.2" targetFramework="net45" />
+  <package id="Microsoft.Data.OData" version="5.8.4" targetFramework="net452" />
```
Comment (Pilchie) [inline test/Microsoft.AspNet.WebHooks.Custom.AzureStorage.Test/packages.config:6]: Same about the 452 change here?

**Review by Pilchie — APPROVED** [review]

**Comment by wtgodbe** [comment]
Got a bunch of consistent test failures in AzureStorage tests now after updating the OData dependency:

> Hooks.Custom.AzureStorage\Storage\StorageManager.cs(121): error : Microsoft.AspNet.WebHooks.Storage.StorageManagerTests.ExecuteDeleteAll_HandlesEmptySet: System.InvalidOperationException : Could not initialize connection to Microsoft Azure Storage: \r\n---- Microsoft.WindowsAzure.Storage.StorageException : Could not load file or assembly 'Microsoft.Data.OData, Version=5.6.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies. The system cannot find the file specified.\r\n-------- System.IO.FileNotFoundException : Could not load file or assembly 'Microsoft.Data.OData, Version=5.6.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35' or one of its dependencies. The system cannot find the file specified. [D:\a\1\s\tools\WebHooks.xunit.targets]
D:\a\1\s\Src\Microsoft.AspNet.WebHooks.Custom.AzureStorage\Storage\StorageManager.cs(121): error :    at Microsoft.AspNet.WebHooks.Storage.StorageManager.<>c__DisplayClass12_0.<GetCloudTable>b__0(String key) in D:\a\1\s\Src\Microsoft.AspNet.WebHooks.Custom.AzureStorage\Storage\StorageManager.cs:line 121 [D:\a\1\s\tools\WebHooks.xunit.targets]
D:\a\1\s\Src\Microsoft.AspNet.WebHooks.Custom.AzureStorage\Storage\StorageManager.cs(121): error :    at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory) [D:\a\1\s\tools\WebHooks.xunit.targets]
D:\a\1\s\Src\Microsoft.AspNet.WebHooks.Custom.AzureStorage\Storage\StorageManager.cs(121): error :    at Microsoft.AspNet.WebHooks.Storage.StorageManager.GetCloudTable(String connectionString, String tableName) in D:\a\1\s\Src\Microsoft.AspNet.WebHooks.Custom.AzureStorage\Storage\StorageManager.cs:line 104 [D:\a\1\s\tools\WebHooks.xunit.targets]
D:\a\1\s\Src\Microsoft.AspNet.WebHooks.Custom.AzureStorage\Storage\StorageManager.cs(121): error :    at Microsoft.AspNet.WebHooks.Storage.StorageManagerTests.InitializeTable() in D:\a\1\s\Test\Microsoft.AspNet.WebHooks.Custom.AzureStorage.Test\Storage\StorageManagerTests.cs:line 481 [D:\a\1\s\tools\WebHooks.xunit.targets]
D:\a\1\s\Src\Microsoft.AspNet.WebHooks.Custom.AzureStorage\Storage\StorageManager.cs(121): error :    at Microsoft.AspNet.WebHooks.Storage.StorageManagerTests.<ExecuteDeleteAll_HandlesEmptySet>d__31.MoveNext() in D:\a\1\s\Test\Microsoft.AspNet.WebHooks.Custom.AzureStorage.Test\Storage\StorageManagerTests.cs:line 0 [D:\a\1\s\tools\WebHooks.xunit.targets]

Can't load 5.6.2.0. Will check where we're still referencing that

**Comment by dougbu** [comment]
> Can't load 5.6.2.0. Will check where we're still referencing that

Suspect the offending project or projects references WindowsAzure.Storage. Might want to try moving to Azure.Storage.Blobs. See https://github.com/dotnet/aspnetcore/commit/220d4d14e05e15b4cd8a19a0da23c88a90ccd37d for another place I had to make that change.

**Comment by wtgodbe** [comment]
Ok, finally got this working. Forgot about binding redirects 🤦

**Pull request merged**